### DISCUSSION TITLE
Mni2subject_coords Error

### DISCUSSION THREAD

**Discussion by gruberanthony** [simnibs/simnibs#301] [9. Results and analysis]
Attempting to run a ROI analysis of the J values from a simulation but I'm running into an error when attempting to apply MNI coordinates. 
There was an error running mni2subject_coords:
 /apps/simnibs/4.0.0/simnibs_env/lib/python3.9/site-packages/requests/__init__.py:102: RequestsDependencyWarning: urllib3
 (1.26.15) or chardet (5.2.0)/charset_normalizer (2.0.4) doesn't match a supported version!
  warnings.warn("urllib3 ({}) or chardet ({})/charset_normalizer ({}) doesn't match a supported "

Not sure how of if I should change these libraries to make things work.

**Comment by oulap** [comment]
Did you install simnibs using the installer, or did you create the environment yourself? The error sounds like you might be mixing some python package versions.

**Reply by gruberanthony** [reply]
To my knowledge the install of simnibs is from the installer. However, I am using UF's HiPerGator so there could be additional intricacies I'm unaware of.
